### PR TITLE
Tower of hanoi

### PR DESCRIPTION
**Pull request by gaara4896**

### CODE REVIEW

File: code/unclassified/TowerOfHanoi/TowerOfHanoi.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import scala.collection.mutable.ArrayStack
```
Comment (yatharthshahjpr) [inline code/unclassified/TowerOfHanoi/TowerOfHanoi.scala]: Add  this comment on the top of every code you submit  : -
/* Part of Cosmos by OpenGenus Foundation */     
👍

File: code/unclassified/average/average.scala [hunk 2]
```diff
@@ -0,0 +1,9 @@
+object Average extends App{
+	def average(num:Array[Double]):Double = {
+		val sum:Double = num.foldLeft(0.0){(a, b) => a + b}
+		sum / num.size
+	}
+
+	val arr:Array[Double] = Array(1, 2, 3, 4, 5)
+	println(average(arr))
+}
```
Comment (yatharthshahjpr) [inline code/unclassified/average/average.scala]: Please add a new line after the code ends ! 👍

Comment (AdiChat) [reply code/unclassified/average/average.scala]: Any technical reason for this?

Comment (yatharthshahjpr) [reply code/unclassified/average/average.scala]: Some compilers (especially C or C++ ones) give you warnings about:

"No new line at end of file"

Comment (AdiChat) [reply code/unclassified/average/average.scala]: Scala is not C or C++. Moreover,  we do not see a new line at the end of file in your pull requests containing C code 👍

**Comment by AdiChat** [comment]
Kindly move the tower of hanoi code to `mathematical-algorithms/tower_of_hanoi/` and this will be good too merge 👍

File: code/unclassified/average/average.scala (hunk 2 above)
Comment (arnavb) [reply code/unclassified/average/average.scala]: @AdiChat It is required by the C and Pre C++11 standard that there is a newline at the end of every source file ([Source](https://stackoverflow.com/a/8172281/6525260))

**Pull request closed without merging**